### DISCUSSION TITLE
New ConanCenter website is live

### DISCUSSION THREAD

**Discussion by prince-chrismc** [conan-io/conan-center-index#19010] [Announcements]
We are super excited to be announcing the completely new ConanCenter website https://conan.io/center/. With well over a year into its development, it’s been a lot of work and love to get it ready. Why now? Well, the old one stopped updating. So in order to provide continuity we made the decision to go live with this. 

#### WE NEED YOU FEEDBACK! 📣

We put a ton of focus and effort into making the new backend as flexible as possible, and this really means the sky's the limit to what YOU can imagine. We are actively trying to gather your feedback on how the ConanCenter website could help with your daily work! 

The new website is on GitHub and we strongly [recommend you open issues](https://github.com/conan-io/web/issues) with any bugs, feedback, feature requests, or design ideas you have! We want you to help shape the workflows it enables.

Special thanks to @davidsanfal and @franramirez688, two names which you might not see often but deserve a huge 👏 as they have been super hard at work behind the scenes.

**Comment by 13steinj** [comment]
While I love the conan project, this appears as if you made a new website in a locked room, and now are forcing it upon everyone.

Which, I guess, is technically within your right... but it's not a good way to interact with the community / your users.

I'm going to be perfectly honest, I'm not a fan. Both of the general redesign (but maybe I'm just too old or something) and of the lack of information.

The old version of the site provided a lot more information (such as the recipe!) that was usable **at a glance** which I can now only look at either from this repository, and I'll have to **dig**, or from the command line, which means I'll have to **dig**.

I think the worst part is the redesigned website actively encourages people not to use it in terms of it's UX. The searchbar doesn't provide a live search with suggestions; which I could have sworn the old site did.

But furthermore, I don't think forcing this upon people was the right move. It would definitely been nice to have some announcement of the fact that this is going to be redone, that there's a beta site, link it on the current (now dead) site, and actively take feedback _before_ pushing this out.

**Reply by kammce** [reply]
I like the change BUUUUT, we lost a lot of functionality. I think I can do without the recipe embedded in the page. We have a link and thats good enough for me. But a TON of information from the last page is lost. If we could add back those old features I think this would be great. 

Also I really don't like how this is styled for the version. 
<img width="1108" alt="image" src="https://github.com/conan-io/conan-center-index/assets/4549768/3f587cc4-b352-45c6-8b5e-7b5f40e9e094">. Its not easy to read, especially given the comma's small size. I think having a section at the bottom where you can see all of the version. PyPi does a good job of this in my opinion, but maybe others have differing opinions.

**Reply by jcar87** [reply]
Hi @13steinj  - thank you for your feedback. The backend for the old website was based on a Conan 1.x architecture and over time had become a lot harder to maintain. The team has spent considerable time on a new architecture that is based on Conan 2.0 - this not only simplifies our ability to develop and maintain it, but we can also be far more responsive in addressing user feedback like yours. The team has already addressed multiple issues reported in the past several days, and will continue to do so. We are now able to do multiple releases of the website in the same day, which was very challenging in the past.

We apologize this has caused inconvenience - our goal for the website is to be only informative and to provide information that can already be obtained with the Conan client, so we expect it should be possible to operate even if the website is not available. Our plan is to continue enhancing the functionality of the website based on user feedback, with a commitment of reacting much quicker than before.

> The old version of the site provided a lot more information (such as the recipe!) that was usable at a glance which I can now only look at either from this repository, and I'll have to dig, or from the command line, which means I'll have to dig.

Regarding being able to glance at the recipe, most recipes in the repository range from 100-500 lines, so we would like to understand exactly what information users are looking for in the recipe files themselves - rather than the metadata that we can extract from the recipe or the package info, which is what we show on the website.

Any feedback in this regard will be appreciated so that we can best tailor the website to show the right information. Expecting users to dig through and parse through several versions of 100-500 line `conanfile.py` files is not something we expect the average user of the website to do. Arguably, that could also be described as **digging** through a file - when perhaps the website could conveniently show more information instead. 

> The searchbar doesn't provide a live search with suggestions; which I could have sworn the old site did.

This will be addressed.

**Reply by jcar87** [reply]
Hi @kammce  - we are actively going through the feedback received and this will be addressed. I have opened a new issue to track this.
https://github.com/conan-io/web/issues/72

**Comment by nrbnlulu** [comment]
I can't find Qt in the recipe list

**Reply by jcar87** [reply]
Hi @nrbnlulu - you can find Qt here:
https://conan.io/center/recipes/qt

If you could provide which steps you followed which resulted in not finding Qt, we would hugely appreciate if you could raise it as an issue here: https://github.com/conan-io/web/issues

Thanks!

**Reply by nrbnlulu** [reply]
Whoops I can find it now :monkey_face:
